todoapp/todos/serializers.py

```
@@ -33,3 +33,22 @@ class Meta(TodoBaseSerializer.Meta):
 
     def get_status(self, obj):
         return "Done" if obj.done else "To Do"
+
+
+class TodoCreateSerializer(TodoBaseSerializer):
+    todo = serializers.CharField(write_only=True, source="name")
+    name = serializers.CharField(read_only=True)
+    user_id = serializers.PrimaryKeyRelatedField(
+        queryset=get_user_model().objects.all(), write_only=True, source="user"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)